curator-x-async/src/main/java/org/apache/curator/x/async/api/AsyncCuratorFrameworkDsl.java

```diff
@@ -112,4 +112,11 @@
      * @return builder object
      */
     AsyncRemoveWatchesBuilder removeWatches();
+
+    /**
+     * Start an add watch builder
+     *
+     * @return builder object
+     */
+    AsyncWatchBuilder addWatch();
```

Comment:
Shouldn't addWatch() be named watchers(), line in CuratorFramwork, to be consistent with the naming?

Reply:
No, the DSL verbs are different in for async and sync